core/src/main/java/jenkins/util/AtmostOneTaskExecutor.java

```
@@ -100,6 +107,7 @@ public Void call() throws Exception {
                     try {
                         inprogress.set(task.call());
                     } catch (Throwable t) {
+                        LOGGER.log(Level.WARNING, null, t);
```

…that exception was being completely swallowed…